Src/coffeescript/cql-execution/src/datatypes/interval.coffee

```diff
@@ -7,60 +7,60 @@ cmp = require '../util/comparison'
 module.exports.Interval = class Interval
   constructor: (@low, @high, @lowClosed = true, @highClosed = true) ->
 
-  contains: (item) ->
+  contains: (item, precision) ->
     if item instanceof Interval then throw new Error("Argument to contains must be a point")
     closed = @toClosed()
     ThreeValuedLogic.and(
-      cmp.lessThanOrEquals(closed.low, item),
-      cmp.greaterThanOrEquals(closed.high, item)
+      cmp.lessThanOrEquals(closed.low, item, precision),
+      cmp.greaterThanOrEquals(closed.high, item, precision)
     )
 
-  properlyIncludes: (other) ->
+  properlyIncludes: (other, precision) ->
     if not (other instanceof Interval) then throw new Error("Argument to properlyIncludes must be an interval")
     ThreeValuedLogic.and(
-      @includes(other),
-      ThreeValuedLogic.not(cmp.equals(@, other))
```

Comment:
Believe it or not, it's essentially logically equivalent.  The spec describes "properly includes" as:
> The properly includes operator for intervals returns true if the first interval completely includes the second and the first interval is strictly larger than the second.

Before, we tested `a properly includes b` by checking if `a includes b` first.  If true, then to ensure that `a` is a larger interval than `b`, we checked to be sure they weren't equal.  If `a includes b` but `a != b`, then `a` must be larger than `b`.

For this next part, keep in mind that normal `includes` allows for intervals that are the same.  In other words, `Interval[1,5] includes Interval[1,5]` is `true`...

In the new logic, we still check that `a includes b`.  If it does, rather than check equality, we check to ensure that `b includes a` is `false`.  Because if `b includes a` is `true`, and `a includes b` is also true, then they are effectively equal (at the requested precision, anyway).  This is why the new code is essentially logically equivalent.  If `a includes b` but `not b includes a`, then we know `a` is larger than `b`.

The reason I didn't keep the `cmp.equals` logic is because that `equals` function doesn't support taking precision into account.